include/game/game.h

```diff
@@ -692,14 +690,35 @@ extern void sub_8021350(void);
 // NOTE: Proc type should be the same as SetStageSpawnPosInternal!
 extern void SetStageSpawnPos(u32 character, u32 level, u32 p2, Player *player);
 
+#define INCREMENT_SCORE_A(incVal)                                                       \
+    {                                                                                   \
+        s32 divResA, divResB;                                                           \
+        s32 oldScore = gLevelScore;                                                     \
+        gLevelScore += incVal;                                                          \
+                                                                                        \
+        divResA = Div(gLevelScore, 50000);                                              \
+        divResB = Div(oldScore, 50000);                                                 \
+                                                                                        \
+        if ((divResA != divResB) && (gGameMode == GAME_MODE_SINGLE_PLAYER)) {           \
+            u16 lives = divResA - divResB;                                              \
+            lives += gNumLives;                                                         \
+                                                                                        \
+            gNumLives = ({                                                              \
```

Comment:
Isn't this just `gNumLives = MIN(lives, 255)`?

Reply:
That sadly doesn't match.